x-pack/agent/pkg/agent/transpiler/rules.go

```diff
@@ -148,6 +151,11 @@ func (r *RuleList) UnmarshalYAML(unmarshal func(interface{}) error) error {
 			if err := unpack(fields, r); err != nil {
 				return err
 			}
+		case "extract_list_items":
+			r = &ExtractListItemRule{}
+			if err := unpack(fields, r); err != nil {
+				return err
+			}
```

Comment:
looking at the method, its a problem with all the cases statement of the switch case, we could do:

```suggestion
			return unpack(fields, r)
```

Reply:
we cant return just but i will change it to less chatty way